Tasks/DotNetCoreInstallerV1/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 0,
-        "Patch": 4
+        "Minor": 1,
```

Comment:
Himanshu is also changing this. Check the version.

Reply:
Yeah, I have to check the version of all the modified tasks, as many of them have been upgraded since I have written this code.